13_3_Setting_Up_a_Channel.md

```diff
@@ -0,0 +1,230 @@
+* Open channel.
+
+#### Fund you c-lightning wallet.
+
+> :book: ***What is a c-lightning wallet?*** 
+
+C-lightning standard implementation comes with a integrated bitcoin wallet that allows you send and receive bitcoin onchain transactions.  This wallet will be used to create new channels.   The first thing you need to do is send some satoshis to your c-lightning wallet.  You can create a new address using  `lightning-cli newaddr` command to use it later. The newaddr RPC command generates a new address which can subsequently be used to fund channels managed by the c-lightning node.  This last transaction is called the [funding transaction](https://github.com/lightningnetwork/lightning-rfc/blob/master/03-transactions.md#funding-transaction-output) and it needs to be confirmed before funds can be used.  You can specify the type of address wanted,  if not specified the address generated will be a bech32.
+
+```
+c$ lightning-cli --network=testnet newaddr
+{
+   "address": "tb1qefule33u7ukfuzkmxpz02kwejl8j8dt5jpgtu6",
+   "bech32": "tb1qefule33u7ukfuzkmxpz02kwejl8j8dt5jpgtu6"
+}
+```       
+We send some sats to this address in this transaction [11094bb9ac29ce5af9f1e5a0e4aac2066ae132f25b72bff90fcddf64bf2feb02](https://blockstream.info/testnet/tx/11094bb9ac29ce5af9f1e5a0e4aac2066ae132f25b72bff90fcddf64bf2feb02)
+
+To check you local balance you should use `lightning-cli listfunds` command:
+
+```       
+c$ lightning-cli --network=testnet listfunds
+{
+   "outputs": [],
+   "channels": []
+}
+```       
+
+Since we still do not have 6 confirmations we do not have balance available,  after 6 confirmations we should see balance:
+
+```       
+c$ lightning-cli --network=testnet listfunds
+{
+   "outputs": [
+      {
+         "txid": "11094bb9ac29ce5af9f1e5a0e4aac2066ae132f25b72bff90fcddf64bf2feb02",
+         "output": 0,
+         "value": 300000,
+         "amount_msat": "300000000msat",
+         "scriptpubkey": "0014ca79fcc63cf72c9e0adb3044f559d997cf23b574",
+         "address": "tb1qefule33u7ukfuzkmxpz02kwejl8j8dt5jpgtu6",
+         "status": "confirmed",
+         "blockheight": 1780680,
+         "reserved": false
+      }
+   ],
+   "channels": []
+}
+
+```       
+
+Now that we have funded our c-lightning wallet we will get information about remote node to start creating channel process. 
+
+#### Connect to remote node
+
+The first thing you need to do is connect your node to a peer. This is done with the `lightning-cli connect` command. Remember that if you want more information on this command, you should type `lightning-cli help connect`.   The connect RPC command establishes a new connection with another node in the Lightning Network.
+
+To connect your node to a remote peer you need it's id that represents the target node’s public key. As a convenience, id may be of the form id@host or id@host:port.  Using `lightning-cli listnodes` command you obtain all nodes available on the network and choose one.
+
```

Comment:
We do need to tell people how to set up that remote node, however they do it. The goal with these tutorials is always to present the real opportunity to work through everything and try out all the commands, and here we have a gap. So it might be as simple as saying "set up a Lightning node on a second machine using the Standup script". I just want to make it clear for people that it's a step.

Reply:
Done.